builtin/logical/postgresql/backend_test.go

```diff
@@ -228,7 +250,7 @@ func testAccStepReadCreds(t *testing.T, b logical.Backend, name string) logicalt
 			}
 
 			userRows := returnedRows()
-			if userRows != 2 {
+			if userRows < 2 {
```

Comment:
Can this be made a constant and given a description for readability?  This is just a test, but the same can be applied elsewhere.

```
// minNumPermissions is the minimum number of permissions that will always be present.
const minNumPermissions = 2
```